dc/council/code/titles/16/sections/16-3902.xml

```diff
@@ -21,55 +21,62 @@
-      <text>[seal] __________________</text>
-      <text>Clerk of the Small Claims and</text>
-      <text>Conciliation Branch, Superior Court of the District of Columbia</text>
-    </para>
+    <text>
+      <center>
+        <strong>SUPERIOR COURT OF THE DISTRICT OF COLUMBIA</strong>
+        <br/>
+        <strong>SMALL CLAIMS AND CONCILIATION BRANCH</strong>
+      </center>
+    </text>
+    <text>(Location of room in  (Address of</text>
+    <text>courthouse)  court)</text>
+    <text>
+      <center>
+        <strong>Washington, D.C.</strong>
+      </center>
+    </text>
+    <text>__________________</text>
+    <text>Plaintiff</text>
+    <text>__________________</text>
+    <text>Address  No. _______</text>
+    <text>
+      <em>vs.</em>
+    </text>
+    <text>__________________</text>
+    <text>Defendant</text>
+    <text>
+      <center>
+        <strong>STATEMENT OF CLAIM</strong>
+      </center>
+    </text>
+    <text>(Here the plaintiff, or at his request the clerk, will insert a statement of the plaintiff’s claim, and the original, to be filed with the clerk, may, if action is on a contract, express or implied, be verified by the plaintiff or his agent, as follows:)</text>
+    <text>District of Columbia,<em>ss:</em></text>
+    <text>____________ being first duly sworn on oath says the foregoing is a just and true statement of the amount owing by defendant to plaintiff, exclusive of all set-offs and just grounds of defense.</text>
+    <text>__________________.</text>
+    <text>Plaintiff (or agent)</text>
+    <text>Subscribed and sworn to before me this ______ days of _______, 19__.</text>
+    <text>__________________.</text>
+    <text>Clerk (or notary public)</text>
+    <text>
+      <center>
+        <strong>NOTICE</strong>
+      </center>
+    </text>
+    <text>To: ____________</text>
+    <text>Defendant</text>
+    <text>____________</text>
+    <text>Home address</text>
+    <text>____________</text>
+    <text>Business address</text>
+    <text>You are hereby notified that ____________ has made a claim and is requesting judgment against you in the sum of _______ dollars ($__), as shown by the foregoing statement. The Court will hold a hearing upon this claim on ____________ at __m. in the Small Claims and Conciliation Branch (address of Court).</text>
+    <text>You are required to be present at the hearing in order to avoid a judgment by default.</text>
+    <text>If you have witnesses, books, receipts, or other writings bearing on this claim, you should bring them with you at the time of the hearing.</text>
+    <text>If you wish to have witnesses summoned, see the clerk at once for assistance.</text>
+    <text>If you admit the claim, but desire additional time to pay, you must come to the hearing in person and state the circumstances to the court.</text>
+    <text>You may come with or without an attorney.</text>
+    <text>[seal] __________________</text>
```

Comment:
'seal' in the word document is small capped not lower case (Division II, pg 729).